src/Components/Common/UserAutocompleteFormField.tsx

```diff
@@ -148,11 +149,21 @@ export const LinkedFacilityUsers = (props: LinkedFacilitySearchProps) => {
   );
 };
 
-const userOnlineDot = (user: UserBareMinimum) => (
-  <div
-    className={classNames(
-      "mr-4 size-2.5 rounded-full",
-      isUserOnline(user) ? "bg-primary-500" : "bg-secondary-400",
-    )}
-  />
-);
+const userOnlineDot = (user: UserBareMinimum) => {
+  console.log(user);
+  return (
+    <div className="mr-2 flex items-center">
+      <div
+        className={classNames(
+          "mr-4 size-2.5 rounded-full",
+          isUserOnline(user) ? "bg-primary-500" : "bg-secondary-400",
+        )}
+      />
+      <Avatar
+        name={formatName(user)}
+        imageUrl={user.profile_picture}
```

Comment:
`profile_picture`? Are you sure this is correct?

It's not working. The user shown below has a profile picture uploaded, but not shown here. Please test your PR :) 

<img width="975" alt="image" src="https://github.com/user-attachments/assets/d25ff73d-b4a4-4520-94a1-44d0e319a349">

Reply:
How to get Rithvik Nishad as a suggestion? When typed it is showing no option found

Reply:
You can try logging in to the doctor credentials provided in the README and upload a profile pic for that user, and try to create a consultation for any patient.